Source/Scene/ClippingPlaneCollection.js

```diff
@@ -66,6 +73,30 @@ define([
      * @param {Boolean} [options.unionClippingRegions=false] If true, a region will be clipped if included in any plane in the collection. Otherwise, the region to be clipped must intersect the regions defined by all planes in this collection.
      * @param {Color} [options.edgeColor=Color.WHITE] The color applied to highlight the edge along which an object is clipped.
      * @param {Number} [options.edgeWidth=0.0] The width, in pixels, of the highlight applied to the edge along which an object is clipped.
+     *
+     * @demo {@link https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/?src=3D%20Tiles%20Clipping%20Planes.html|Clipping 3D Tiles and glTF models.}
+     * @demo {@link https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/?src=Terrain%20Clipping%20Planes.html|Clipping the Globe.}
+     *
+     * @example
+     * // This clipping plane's distance is positive, which means its normal
+     * // is facing the origin. This will clip everything that is behind 
+     * // the plane, which is all pixels with y > 5.
```

Comment:
I initially said something like "all points with y > 5" but that to me implies like clipping vertices, and saying "objects" implies whole objects are hidden/shown. "Pixels" is really what gets clipped, but I changed it to "anything with y coordinate > 5" since all we want to communicate here is in what direction the clipping happens.